### DISCUSSION TITLE
Welcome to Beaker discussions!

### DISCUSSION THREAD

**Discussion by pfrazee** [beakerbrowser/beaker#1548] [General] [answered]
Please feel free to ask questions, discuss ideas, and share anything Beaker related!

**Comment by danrobi11** [comment, marked as answer]
Nice! Well lets start with a big thank you for beaker :)

**Comment by danass** [comment]
Any hyper:// links sharing platform?

**Comment by danrobi11** [comment]
@pfrazee Regarding your [decentralization and decency talk](https://invidious.site/watch?v=ULFj714_Vvo)

About something like mastodon. Users would simply moderate their channel.

**Comment by pfrazee** [comment]
Check out https://github.com/pfrazee/ctzn which is where I’m currently spending my time. I’ve been streaming development on YouTube

**Reply by danrobi11** [reply]
Thanks for the ctzn link. I knew about ctzn.
ctzn is not really in the beaker spirit. Install a server and get a dns domain name ?
Thats exactly why I use beaker and ipfs is to avoid all that.
Maybe open a ctzn discussion? Im banned on twitter.

**Comment by odarcan** [comment]
Does beaker allow for the user to browse all online hyperdrives?
